findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
:ant: this should not be necessary. Try deleting it, and the `findbugs-maven-plugin` configuration override.